src/main/java/com/training/web/pages/flipkart/FlipkartHomePage.java

```diff
@@ -25,4 +28,32 @@ public ResultPage clickSearch() {
         $x(submit).shouldBe(visible).click();
         return new ResultPage();
     }
+
+    public String clickSocialMediaPage(String media) throws InterruptedException {
+
+        switch (media) {
+            case "Facebook":
+                SelenideElement facebook = $x("//a[text()='Facebook']");
```

Comment:
Use variables instead xpath ,initialize this globally ,Apply this changes for below xpath also